deployment/refinery-modules/refinery/manifests/postgresql.pp

```diff
@@ -1,5 +1,7 @@
 class refinery::postgresql {
-  $server_version = '9.3'
+  # Postgres versioning scheme provides latest point releases when
+  # specifying major version: https://www.postgresql.org/support/versioning/
+  $server_version = '10'
```

Comment:
The minor version should be specified explicitly. If a new instance gets a new minor version which does not match the version of the snapshot then restore will fail.

Reply:
Specifying the `10` here isn't any different than how we were specifying the `9.3` before. Those are both major versions of PostgreSQL. It just happens to be that the Postgres folks have changed their versioning scheme starting with the [`10` series of releases](https://www.postgresql.org/support/versioning/).

Reply:
OK, fair enough. This is not the RDS version actually.